utils/flink-components-testkit/src/test/scala/pl/touk/nussknacker/engine/flink/util/test/FlinkTestScenarioRunnerSpec.scala

```diff
@@ -130,8 +131,15 @@ class FlinkTestScenarioRunnerSpec extends AnyFunSuite with Matchers with FlinkSp
           collector: ServiceInvocationCollector,
           componentUseCase: ComponentUseCase
       ): Future[String] = {
-        collector.collect(s"test-service-$value", Option(MockedValued)) {
-          Future.successful(value.evaluate(context))
+        componentUseCase match {
```

Comment:
I'm not sure about this change.. ServiceInvocationCollector is responsible for collect that and potential mockup -> checkout this api.